tests/test_generate.py

```diff
@@ -130,6 +130,25 @@ def test_generator_create_widget_keyerror(generator, caplog):
     )
 
 
+def test_generator_create_widget_is_list_of_dicts(generator):
+    generator._get_screen_dimensions = Mock(return_value=(800, 1280))
+    generator._is_list_of_dicts = Mock(return_value=True)
+    generator.screen_name = "test"
+    component = Entity(
+        type="ADAravis.aravisCamera", P="BL23B-DI-MOD-02", desc=None, M=None, R="CAM:"
+    )
+    generator._allocate_widget = Mock(
+        return_value=Widget.EmbeddedDisplay(
+            name="X", file="", x=0, y=0, width=205, height=120
+        )
+    )
```

Comment:
Can you rearrange these so that the mocked functions are all above the variables?

```suggestion
    generator._get_screen_dimensions = Mock(return_value=(800, 1280))
    generator._is_list_of_dicts = Mock(return_value=True)
    generator._allocate_widget = Mock(
        return_value=Widget.EmbeddedDisplay(
            name="X", file="", x=0, y=0, width=205, height=120
        )
    )

    generator.screen_name = "test"
    component = Entity(
        type="ADAravis.aravisCamera", P="BL23B-DI-MOD-02", desc=None, M=None, R="CAM:"
    )

```